__tests__/app.test.js

```diff
@@ -72,9 +75,9 @@ describe( "/api/articles/:article_id", ()=> {
               });
           });
 
-    it("GET 404 when given article_id is not found", () => {
+    it("GET 404 when given article_id is not found in database", () => {
```

Comment:
These small changes to test descriptions make it much easier to understand what you are testing for, good work!